spec/factories/applications.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :application do
+    sequence(:name) { |n| "app_#{n}" }
+    sequence(:url) { |n| "#{n}#{n}#{n}#{n}.com" }
```

Comment:
How else could the string interpolation `"#{n}#{n}#{n}#{n}.com"` be written if the solution had to use `* 4` as part of the code?